packages/css/src/components/avatar/README.md

```diff
@@ -6,10 +6,11 @@ A circular badge representing a person.
 
 ## Design
 
-The avatar contains 1 or 2 initial letters from the person's full name, a picture, or a generic icon.
-The default background colour is dark blue.
+The avatar contains 1 or 2 initial letters from the person’s full name, a picture, or a generic icon.
+To personalize the avatar, the user can be allowd to choose one of the highlight colours.
+The default background colour is purple.
 
 ## Guidelines
 
-- Display an avatar for the person currently using the application,
+- Display an avatar for the person who is using the application,
```

Comment:
```suggestion
- Show an avatar for the user of the application,
```

(Sidenote: I often use [this AI text rewriter](https://ahrefs.com/writing-tools/paragraph-rewriter) on the 'Simplified' setting, that generates pretty good results imo.)